node/node.go

```diff
@@ -179,12 +178,9 @@ func New(cfg *Config, version string) (*Node, error) { //nolint:gocyclo,funlen
 	}
 
 	if cfg.P2P {
-		var privKeyStr string
-		privKeyStr, _ = os.LookupEnv("P2P_PRIVATE_KEY")
-		var p2pService *p2p.Service
-		p2pService, err = p2p.New(cfg.P2PAddr, "juno", cfg.P2PBootPeers, privKeyStr, cfg.Network, log)
+		p2pService, err := p2p.New(cfg.P2PAddr, "juno", cfg.P2PBootPeers, "", cfg.Network, log)
 		if err != nil {
-			return nil, fmt.Errorf("set up p2p service: %w", err)
+			return nil, fmt.Errorf("Failed to set up p2p service: %w", err)
```

Comment:
nit: I think the previous error message is more idiomatic.

Reply:
Reverting. Could you please send a reference right now we have a mixture of these in the codebase.